docs/index.rst

```diff
@@ -6,15 +6,20 @@ Solidity
     :alt: Solidity logo
     :align: center
 
-Solidity is a contract-oriented, high-level language for implementing smart contracts.
-It was influenced by C++, Python and JavaScript
-and is designed to target the Ethereum Virtual Machine (EVM).
+Solidity is a contract-oriented, high-level language for implementing smart
+contracts. Smart contracts are programs which govern the behaviour of accounts
```

Comment:
I agree this may be more complex. I would also propose to change `is a contract-oriented, high-level language` to` is a Ethereum oriented, high-levle language`. Not sure `contract` is explained at this stage.

Reply:
this is meant as a pun on "object oriented" - we can also just use "object oriented".

Reply:
@axic and @chriseth, In all honesty, I was a little uncertain about re-introducing some of this content anyway, my only feeling was that the pre-existing content didn't really tell you what you could do with Solidity enough. Maybe this is a task better handled at a higher level from the Ethereum project website, but people arrive at pages from search engines etc and may have never read that.

That said, try these changes that merge some of your suggestions (or we just drop the changes). And I am also preparing another PR to improve this page further once this PR is merged.

Reply:
> Well perhaps then state-oriented is better? :)

@chriseth there was no response to this yet